orchestrators/ProcurementOperations/tests/Unit/Services/VendorSelectionServiceTest.php

```
@@ -0,0 +1,496 @@
+
+        $this->assertFalse($result['qualifies']);
+        $this->assertCount(4, $result['reasons']);
+    }
+
+    private function assertFloatEquals(float $expected, float $actual, float $delta): void
+    {
+        $this->assertTrue(
+            abs($expected - $actual) <= $delta,
+            sprintf('Expected %.2f to equal %.2f within delta %.2f', $actual, $expected, $delta)
+        );
+    }
```

PHPMD (2.15.0)

[warning] Avoid unused private methods such as 'assertFloatEquals'. (undefined)

(UnusedPrivateMethod)